server/controllers/availability.js

```diff
@@ -0,0 +1,88 @@
+const Availability = require("../models/Availability");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+
+// @route POST /availability
+// @desc create a schedule
+// @access private
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+    const schedule = req.body.schedule
+   
+    let newSchedule = new Availability({
+      profileId: schedule.profileId,
+      schedule: schedule.schedule
+    })
+
+    let savedSchedule = await newSchedule.save()
```

Comment:
```suggestion
    const savedSchedule = await newSchedule.save()
```
Same as comment above